src/emitter.js

```diff
@@ -126,7 +118,7 @@ class Emitter {
   /**
    * Remove all events for a given scope
    */
-  _removeScope(scope) {
+  _removeScope(scope: any) {
```

Comment:
Just pointing out `any` is unsafe. Is there a good reason for using it vs mixed (I think) https://flow.org/en/docs/types/any/